protzilla/constants/paths.py

```diff
@@ -1,8 +1,9 @@
 from pathlib import Path
 
 PROJECT_PATH = Path(__file__).resolve().parent.parent.parent
-RUNS_PATH = Path(PROJECT_PATH, "user_data/runs")
-WORKFLOWS_PATH = Path(PROJECT_PATH, "user_data/workflows")
+USER_DATA_PATH = Path(PROJECT_PATH, "user_data")
+RUNS_PATH = USER_DATA_PATH / "runs"
+WORKFLOWS_PATH = USER_DATA_PATH / "workflows"
 EXTERNAL_DATA_PATH = Path(PROJECT_PATH, "user_data/external_data")
```

Comment:
Wenn du oben die "userdata/..." paths geändert hast, wollen wir das dann in der Zeile hier auch ändern, damit es einheitlich ist?